From b780977375e59988916c537684b2fbc7a7c96c76 Mon Sep 17 00:00:00 2001
From: Steven Webster <steven.webster@windriver.com>
Date: Wed, 28 Feb 2018 10:23:41 -0500
Subject: [PATCH 144/155] CGTS-8968: Add sfc,flow_classifer to default
 service_plugins

Currently, it is possible to overwrite the sfc and flow_classifier
service_plugins by overwriting the associated service parameter
value.

This causes issues with 'get_sfc_details' timeouts on the agent
side, as the plugins are not loaded on the server side.

We require that the sfc and flow_classifier plugins always be
enabled, so we can simply add them to the list of default
neutron service plugins.
---
 neutron/plugins/common/constants.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/neutron/plugins/common/constants.py b/neutron/plugins/common/constants.py
index 7ddd031..efe0d4b 100644
--- a/neutron/plugins/common/constants.py
+++ b/neutron/plugins/common/constants.py
@@ -49,6 +49,8 @@ DEFAULT_SERVICE_PLUGINS = {
     'flavors': 'flavors',
     'revisions': 'revisions',
     'trunk': 'trunk',
+    'sfc': 'sfc',
+    'flow_classifier': 'flow_classifier'
 }
 
 # Service operation status constants
-- 
2.7.4

